identity/tests/sample_data.rs

```diff
@@ -11,7 +11,7 @@ pub fn users() -> Vec<UserAddUpdate> {
             family_name: "Doe".into(),
             picture: "https://example.com/avatar.jpg".into(),
             oauth_access_token: "access_token".into(),
-            oauth_access_token_valid: SystemTime::now() + Duration::from_secs(3600),
+            oauth_access_token_valid: NaiveDate::from_ymd(2020, 12, 31).and_hms(0, 0, 0),
```

Comment:
```suggestion
            oauth_access_token_valid: NaiveDate::from_ymd(2020, 12, 31),
```
Can't you leave the `.and_hms(0, 0, 0)` away?

Reply:
In theory yes, but I prefer it explicit here and also I want to adapt the tests later to more complex scenarios.

Reply:
👍🏾